pkgs/top-level/all-packages.nix

```diff
@@ -12184,12 +12184,6 @@ with pkgs;
 
   meshlab = callPackage ../by-name/me/meshlab/package.nix {
     stdenv = if stdenv.hostPlatform.isDarwin then llvmPackages_18.stdenv else stdenv;
```

Comment:
@yzx9: maybe we can remove that too, right ? And move meshlab to pkgs/by-name ?

Reply:
Yes, we should able to remove the entry from `all-packages.nix`